<?xml version="1.0" encoding="utf-8"?>
<run passed="0" failed="1" unexecuted="0" time="474" total="1">
  <tests>
    <test key=".\SalaryCostApi" name="SalaryCostApi" outcome="failed" time="151">
      <step id="5erTiX6FiI0ZwnWyljT8Sl" name="CreateId" outcome="failed" time="112">
        <action id="2J6tsbc8viWtqwq8MTntZ6" type="coded-step" outcome="failed" time="103">
          <error message="Object reference not set to an instance of an object." />
        </action>
      </step>
    </test>
  </tests>
  <output>
    <output message="The test case is starting ..." />
    <output message="Create Excel Result File" />
    <output message="Time of starting Test :: _24032023_10_36_17" />
    <output message="File Not Find" />
    <output message="Get CSV :: Could not find file 'C:\Users\anchaudhary\Documents\Test Studio Projects\Indus-PerformancesMeasurements\PMD\TestFile\TestPerf.csv'." />
    <output message="Ending the test case ..." />
  </output>
  <errors>
    <error message="Running test's 'OnAfterTestCompleted' method failed. Message: 'Index was outside the bounds of the array.'" />
  </errors>
</run>